packages/svelte/tests/snapshot/samples/inline-module-vars/_expected/client/index.svelte.js

```diff
@@ -1,33 +1,21 @@
 import "svelte/internal/disclose-version";
 
-const __ENHANCED_IMG_1__ = "__VITE_ASSET__2AM7_y_a__";
-const __ENHANCED_IMG_2__ = "__VITE_ASSET__2AM7_y_b__";
-const __ENHANCED_IMG_3__ = "__VITE_ASSET__2AM7_y_c__";
-const __ENHANCED_IMG_4__ = "__VITE_ASSET__2AM7_y_d__";
-const __ENHANCED_IMG_5__ = "__VITE_ASSET__2AM7_y_e__";
-const __ENHANCED_IMG_6__ = "__VITE_ASSET__2AM7_y_f__";
+const __DECLARED_ASSET_0__ = "__VITE_ASSET__2AM7_y_a__ 1440w, __VITE_ASSET__2AM7_y_b__ 960w";
+const __DECLARED_ASSET_1__ = "__VITE_ASSET__2AM7_y_c__ 1440w, __VITE_ASSET__2AM7_y_d__ 960w";
+const __DECLARED_ASSET_2__ = "__VITE_ASSET__2AM7_y_e__ 1440w, __VITE_ASSET__2AM7_y_f__ 960w";
+const __DECLARED_ASSET_3__ = "__VITE_ASSET__2AM7_y_g__";
 
 import * as $ from "svelte/internal/client";
 
-var root = $.template(`<picture><source type="image/avif"> <source type="image/webp"> <source type="image/png"> <img alt="production test" width="1440" height="1440"></picture>`);
+var root = $.template(`<picture><source srcset="${__DECLARED_ASSET_0__}" type="image/avif"> <source srcset="${__DECLARED_ASSET_1__}" type="image/webp"> <source srcset="${__DECLARED_ASSET_2__}" type="image/png"> <img src="${__DECLARED_ASSET_3__}" alt="production test" width="1440" height="1440"></picture>`);
```

Comment:
I think you'll need to insert escape calls here, unless you restrict the inlining only to literals that can be statically evaluated.

Reply:
good idea. I've restricted it to literals for now. I'd like to avoid escaping those so that they can be better minified